package/utils/busybox/Config-defaults.in

```diff
@@ -2239,8 +2248,7 @@ config BUSYBOX_DEFAULT_FEATURE_WATCHDOG_OPEN_TWICE
 	default n
 config BUSYBOX_DEFAULT_FEATURE_IPV6
 	bool
-	default y if IPV6
-	default n
+	default y
```

Comment:
This looks wrong. You have not yet returned the OpenWrt logic here. One of the corrections mentioned in my PR comments.

Reply:
Actually, I thought `CONFIG_IPV6` had been completely dropped, but it seems the symbol still exists. Will fix, thanks.